fhir-model/src/test/java/com/ibm/fhir/model/util/test/ValidationSupportTest.java

```diff
@@ -154,4 +154,160 @@ public void testCheckLanguageCodeableConceptsNotValid() {
         catch (IllegalStateException e) {}
     }
     
+    @Test
+    public void testCheckUcumCodeValid() {
+        ValidationSupport.checkUcumCode(null, "elementName");
+        ValidationSupport.checkUcumCode(Code.of("10.uN.s/(cm.m2)"), "elementName");
+        ValidationSupport.checkUcumCode(Code.of("%{Activity}"), "elementName");
+        ValidationSupport.checkUcumCode(Code.of("[mi_us]"), "elementName");
+    }
+
+    @Test
+    public void testCheckUcumCodeNotValid() {
+        try {
+            ValidationSupport.checkUcumCode(Code.of(null), "elementName");
+            fail();
+        }
+        catch (IllegalStateException e) {}
+        try {
+            ValidationSupport.checkUcumCode(Code.of("{invalid{annotation}}"), "elementName");
+            fail();
+        }
+        catch (IllegalStateException e) {}
+        try {
+            ValidationSupport.checkUcumCode(Code.of("invalid space"), "elementName");
+            fail();
+        }
```

Comment:
style consistency - we merge the catch line with the prior line

Reply:
as a second comment, to this approach, if they are individual tests, then they can succeed and fail independently.

Reply:
There is a code template and formatting rules for Eclipse here (near the bottom of the page):
https://github.com/IBM/FHIR/wiki/Style-Guide

Reply:
Will fix style and break into individual testcases.